bundles/org.eclipse.swt/Eclipse SWT PI/win32/org/eclipse/swt/internal/win32/OS.java

```diff
@@ -4581,4 +4581,12 @@ public static int HRESULT_FROM_WIN32(int x) {
 public static final native boolean DuplicateHandle(long hSourceProcessHandle, long hSourceHandle, long hTargetProcessHandle,
 		long [] lpTargetHandle, int dwDesiredAccess, boolean b, int dwOptions);
 
+
+public static long OpenThemeData(long hwnd, char[] themeName, int dpi) {
+	if (OS.WIN32_BUILD >= OS.WIN32_BUILD_WIN10_1809) {
+		return OS.OpenThemeDataForDpi(hwnd, themeName, dpi);
+	} else {
+		return OS.OpenThemeData(hwnd, themeName);
```

Comment:
Since these 2 methods are now replaced by your new method, it would be worth declaring them `private`.